### DISCUSSION TITLE
InputGroup overflows parent in Firefox

### DISCUSSION THREAD

**Discussion by misterjohannesson** [chakra-ui/chakra-ui#3035] [Q&A] [answered]
Hi guys

I am building a responsive web app and just trying out some things with this lib.

I may have stumbled onto a minor visual bug, or maybe I am using it wrong.

![image](https://user-images.githubusercontent.com/17781340/104378501-61098a00-5528-11eb-8ce2-4ae324b36710.png)

As you can see here, the right addon in the lower input field is outside my boundary, whereas the input above is excellent.

Some simple code to replicate this.
```typescript
  <FormControl isReadOnly>
    <FormLabel>Car id:</FormLabel>
    <Input type="text" value={2} />
  </FormControl>

  <FormControl isInvalid={false} isRequired>
    <FormLabel>Amount filled:</FormLabel>

    <InputGroup>
      <Input
        placeholder="Enter amount filled into tank"
      />
      <InputRightAddon>liters</InputRightAddon>
    </InputGroup>
    <FormErrorMessage>
      Please enter the amount of liters filled into the tank
    </FormErrorMessage>
  </FormControl>
```

I might remedy this by getting the current width attribute from Input above and applying the same to the InputGroup.

Has anyone seen anything like this?
You all think it is a bug I need to open an issue for?

**Comment by dodas** [comment, marked as answer]
Hello @objmj-itminds,
This looks like the same bug: https://github.com/chakra-ui/chakra-ui/issues/3243
Are you using Firefox?
The fix will be out with the next release.

**Reply by misterjohannesson** [reply]
Yes, I am. And it does look like the same bug.

Thank you for letting me know.
